backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/order/inoutcandidate/OrderLineShipmentScheduleHandler.java

```diff
@@ -255,6 +256,9 @@ private void updateShipmentScheduleFromOrderLine(
 		shipmentSchedule.setSinglePriceTag_ID(groupingOrderLineLabel);
 		// 03152 end
 
+		shipmentSchedule.setExternalHeaderId(orderDAO.getExternalId(OrderId.ofRepoId(orderLine.getC_Order_ID())).map(ExternalId::getValue).orElse(null));
```

Comment:
Why here use ExternalId and nowhere else?
This part should be moved to updateShipmentScheduleFromOrder